src/client/core/transport.js

```diff
@@ -13,26 +16,23 @@ export var batchUpdate                     = transport.batchUpdate.bind(transpor
 export var queuedAsyncServiceMsg           = transport.queuedAsyncServiceMsg.bind(transport);
 
 export function fail (err, callback) {
+    // NOTE: we should not stop the test run if an error occured during page unloading because
+    // we destroy the session in this case and we can't get the next page in the browser.
+    // We should set the deferred error to the task to fail the test after the page reloading.
```

Comment:
```
    // NOTE: we should not stop the test run if an error occured during page unloading because
    // we would destroy the session in this case and wouldn't be able to get the next page in the browser.
    // We should set the deferred error to the task to have the test failed after the page reloading.
```